bindings/cpp/include/wtf/macros.h

```diff
@@ -2,6 +2,7 @@
 #define TRACING_FRAMEWORK_BINDINGS_CPP_INCLUDE_WTF_MACROS_H_
 
 #include "wtf/runtime.h"
+#include <thread>
```

Comment:
Believe it or not, we run this on a couple of platforms that do not support C++11 std thread. I'm fine introducing this here and I'll guard it appropriately post-commit -- it would be hard to explain.